### PR TITLE
Add screwdriver build configuration

### PR DESCRIPTION
**Pull request by tkyi**
Configuration enables `npm install` & `npm test`

### CODE REVIEW

**Comment by FenrirUnbound** [comment]
👍 

Looks good to me

t3_4wgvrn

File: screwdriver.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+jobs:
+  main:
+    image: node:6
+    steps:
+      - init: npm install
```
Comment (jer) [inline screwdriver.yaml]: really?

**Comment by nkatzman** [comment]
Probs want to follow what the wercker.yml does https://github.com/screwdriver-cd/guide/blob/master/wercker.yml

**Comment by jer** [comment]
👍

**Comment by nkatzman** [comment]
👍  I hope this works. Would be a sweet demo

**Pull request merged**